source/_posts/2015-11-16-immutable-strings-in-ruby-2-dot-3.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Immutable strings in Ruby 2.3"
+category:
+date: 2015-11-16 08:39:25 -0300
```

Comment:
We're not using this date, just the filename is what the building system uses. This is being generated by the rake task? I'm seeing all the post with it included and unsure where it comes from.

If it's from the rake task please remove that part.